app/models/bookmark.rb

```diff
@@ -21,7 +18,7 @@ class Bookmark < ActiveRecord::Base
 
   scope :my_bookmarks, lambda {
     user = User.current
-    return {} unless SETTINGS[:login] and !user.nil?
+    return all unless SETTINGS[:login] and !user.nil?
```

Comment:
"all" is just an empty scope. scopes can no longer return a hash, they must return a different scope. can be changed to where({}), but I prefer all